ang/civicase/ActivityIcon.html

```diff
@@ -0,0 +1,20 @@
+<span class="civicase__activity-icon" {{activity.icon}}>
+  <span>
+    <span
+      ng-if="!activity.icon && activity.category.indexOf('milestone') < 0"
+      ng-class="{'text-danger': activity.is_overdue}"
+      class="civicase__activity-no-icon">
+      {{activity.type[0]}}
+    </span>
+    <i
+      ng-if="activity.icon"
+      ng-class="{'civicase__activity-icon--ribbon': activity.category.indexOf('milestone') > -1, 'text-danger': activity.is_overdue}"
```

Comment:
Aren't we using material icons here. I feel we should have a standard way of using icon. If we just use material icons we can remove the dependency of font-awesome icons and make new civicase more consistent.

Reply:
I had a discussion with Jon and Shitij about this.(apologies for not conveying it to you). Here the activity icons, will be set by the user itself from the settings. And there we have only font awesome icons. So the same needs to be here.